modules/WebsiteCMS/WebsiteProject/Handlers/DeleteWebsiteProjectHandler.php

```
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Modules\WebsiteCMS\WebsiteProject\Handlers;
+
+use Modules\WebsiteCMS\WebsiteProject\Repositories\WebsiteProjectRepository;
+use Ramsey\Uuid\UuidInterface;
+
+class DeleteWebsiteProjectHandler
+{
+    public function __construct(
+        private WebsiteProjectRepository $repository,
+    ) {
+    }
+
+    public function handle(UuidInterface $id)
+    {
+        $this->repository->deleteWebsiteProject($id);
```

Consider adding error handling around the repository call. What happens if the delete operation fails? The method should either throw an exception or return a boolean status.